sentry/src/test/java/io/sentry/SentryOptionsManipulator.kt

```diff
@@ -0,0 +1,12 @@
+package io.sentry
+
+import io.sentry.clientreport.IClientReportRecorder
+
+class SentryOptionsManipulator {
```

Comment:
Isn't class seems pretty redundant when all we are doing as assigning a property on options?

Reply:
It's used to get around the visibility of clientReportRecorder property